is_core/site.py

```diff
@@ -84,7 +84,7 @@ def get_urls(self):
                     str_to_class(settings.AUTH_LOGOUT_VIEW).as_view(), name='logout'),
             )
 
-        if settings.AUTH_USE_TOKENS:
+        if settings.AUTH_USE_TOKENS and settings.AUTH_RESOURCE_CLASS:
```

Comment:
proc tahle zmena ?

Reply:
protože když nastavím AUTH_RESOURCE_CLASS v settings na None tak nechci aby se resource  přidávalo do urls. Byl bug v is-core